dist/c89-compatible/Hacl_Curve25519_51.c

```diff
@@ -653,7 +654,15 @@ inline static void cswap20(uint64_t bit, uint64_t *p1, uint64_t *p2)
   }
 }
 
-static uint8_t g25519[32U] = { (uint8_t)9U };
+static uint8_t
```

Comment:
you are pushing a snapshot generated with an old version of kremlin; can you update kremlin to minimize the diff?